docs/core/diagnostics/dotnet-trace.md

```diff
@@ -89,7 +89,47 @@ dotnet-trace collect [--buffersize <size>] [--clreventlevel <clreventlevel>] [--
 
 - **`--clrevents <clrevents>`**
 
-  List of CLR runtime events to emit.
+  A `+`-separated list of CLR runtime provider keywords to enable. This is a simple mapping that lets you specify event keywords via string aliases rather than their hex values. For example, `dotnet-trace collect --providers Microsoft-Windows-DotNETRuntime:3:4` requests the same set of events to `dotnet-trace collect --clrevents gc+gchandle`. The table below shows the list of available keywords:
```

Comment:
```suggestion
  A list of CLR runtime provider keywords to enable separated by `+` signs. This is a simple mapping that lets you specify event keywords via string aliases rather than their hex values. For example, `dotnet-trace collect --providers Microsoft-Windows-DotNETRuntime:3:4` requests the same set of events as `dotnet-trace collect --clrevents gc+gchandle`. The table below shows the list of available keywords:
```

Reply:
Also, I see how that specifies the three. How does this notation specify the 4 (verbosity?)